src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -23,7 +23,7 @@
             + " or display a list of all the elderly with incoming visits in the next month (next 30 days) using "
             + " flag " + PREFIX_INCOMING_MONTH
             + " or in the next week (next 7 days) using flag " + PREFIX_INCOMING_WEEK + "\n"
-            + "Parameters: [" + PREFIX_INCOMING_MONTH + "] or [" + PREFIX_INCOMING_WEEK + "]\n"
+            + "Parameters: [CRITERION_FOR_LISTING]\n"
             + "Example: " + COMMAND_WORD + " " + PREFIX_NORMAL_LIST
             + " or " + COMMAND_WORD + " " + PREFIX_INCOMING_MONTH
             + " or " + COMMAND_WORD + " " + PREFIX_INCOMING_WEEK + "\n"
```

Comment:
Double check this, it displays as `list: Displays a list of all the elderly in the system by defaultusing flag all/ or display a list of all the elderly with incoming visits in the next month (next 30 days) using  flag m/ or in the next week (next 7 days) using flag w/` for me. Missing a whitespace at defaultusing

Reply:
Thank you very much for the review!